console/console-init/ui/src/modules/iot-device-detail/components/CredentialsView/CredentialsView.tsx

```diff
@@ -105,15 +100,14 @@ export const Credential: React.FC<ICredentialProps> = ({
         />
         {hasOwnProperty(credential, "enabled") && (
           <>
-            <GridItem
-              span={12}
-              // className={styles.status_section_margin}
-            >
-              Status
+            <GridItem span={12} className={css(styles.status_section_margin)}>
+              <Title headingLevel="h6" size="xl">
```

Comment:
TitleSizes is being used for sizes in the new PF upgrade.

Reply:
Patternfly supports both syntax.